znvis/cameras/particle_following_camera.py

```
@@ -0,0 +1,143 @@
+    2. The camera follows the particle with a dynamic distance vector given by
+       the user. Activated by passing an array of shape (n_frames, 3) to the
+       camera_particle_vector parameter.
+    3. The camera follows the particle while always looking at the same direction
+       as the particle. Activated by passing the particle_directions parameter.
+    """
+
+    def __init__(
+        self,
+        particle_positions: np.ndarray,
+        particle_directions: np.ndarray = None,
+        camera_particle_vector: np.ndarray = np.array([0, 0, 20]),
```

Ruff (0.14.0)

Do not perform function call `np.array` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)